packages/database-version-control/src/migrations/2023.01.26T09.15.49.create-application-session-table.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE IF NOT EXISTS bng.application_session
 (
   application_session_id UUID CONSTRAINT pk_application_session_application_session_id PRIMARY KEY DEFAULT uuid_generate_v4(),
-  application_reference VARCHAR(20) CONSTRAINT uq_application_session_application_reference UNIQUE NOT NULL,
+  application_reference VARCHAR(22) CONSTRAINT uq_application_session_application_reference UNIQUE NOT NULL,
```

Comment:
This needs a new migration to alter the column length as the migration has been merged to master and deployed to cloud environments.